biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/summary/db/Schema.scala

```diff
@@ -0,0 +1,146 @@
+package nl.lumc.sasc.biopet.utils.summary.db
+
+import java.sql.Date
+
+import slick.driver.H2Driver.api._
+
+/**
+ * Created by pjvan_thof on 27-1-17.
+ */
+object Schema {
+
+  case class Run(id: Int, name: String, outputDir: String, version: String, commitHash: String,
```

Comment:
> but I assume this just works.

I would be careful with this. Adding a unit test for this should be possible, right?